analyzers/src/SonarAnalyzer.CSharp/Rules/SillyBitwiseOperation.cs

```diff
@@ -25,6 +25,19 @@ public sealed class SillyBitwiseOperation : SillyBitwiseOperationBase
     {
         protected override ILanguageFacade Language => CSharpFacade.Instance;
 
+        // No codefix (yet?)
+        private const string ComparisonDiagnosticId = "S2198";
+        private const string ComparisonMessageFormat = "Remove this silly mathematical comparison.";
```

Comment:
The suggestion from the rspec is "Remove this comparison; it will always return (false|true)." I would use this or something simpler as the message from `warning CS0652: Comparison to integral constant is useless; the constant is outside the range of type 'byte'`